### DISCUSSION TITLE
nil pointer exception for image in partials/content

### DISCUSSION THREAD

**Discussion by akshaybajaj2280** [thuliteio/doks#930] [Q&A] [answered]
Hello! I'm enjoying doks so far, but ran into a couple issues setting up my website.

I'm using doks version `^0.5` and get nil pointer issues when trying to build. [Here is a link](https://github.com/akshaybajaj2280/akshaybajaj/tree/beta) to my repository.

I suspected that I'm using incompatible dependencies and I also compared it to [doks-child-theme package.json](https://github.com/h-enk/doks-child-theme/blob/master/package.json).

Note that I'm not rendering any images in my content yet. It seems `$image` passes the null check, but fails at [partials/content/card-image.html#13](https://github.com/h-enk/doks/blob/master/layouts/partials/content/figure.html#L11) and [content/card-image.html#L12](https://github.com/h-enk/doks/blob/master/layouts/partials/content/card-image.html#L12) 

I would really appreciate someone's help with this issue. Thank you!

```
npm -version
8.19.2

➜  user git:(beta) npm run server

> doks-child-theme@0.4.3 server
> exec-bin node_modules/.bin/hugo/hugo server

Start building sites …
hugo v0.99.0-1de333e7a3fc863672ec6d6cd53ba66dbcdd2305+extended darwin/arm64 BuildDate=2022-05-16T08:10:56Z VendorInfo=gohugoio
ERROR 2022/10/23 14:32:40 render of "page" failed: "/Users/user/workspace/XXXX/node_modules/@hyas/doks/layouts/blog/single.html:13:9": execute of template failed: template: blog/single.html:13:9: executing "main" at <partial "content/figure.html" .>: error calling partial: "/Users/user/workspace/XXXX/node_modules/@hyas/doks/layouts/partials/content/figure.html:11:18": execute of template failed: template: partials/content/figure.html:11:18: executing "partials/content/figure.html" at <$image.Fill>: nil pointer evaluating resource.Resource.Fill
Error: Error building site: failed to render pages: render of "term" failed: "/Users/user/workspace/XXXX/node_modules/@hyas/doks/layouts/_default/list.html:14:9": execute of template failed: template: _default/list.html:14:9: executing "main" at <partial "content/card-image.html" .>: error calling partial: "/Users/user/workspace/XXXX/node_modules/@hyas/doks/layouts/partials/content/card-image.html:12:18": execute of template failed: template: partials/content/card-image.html:12:18: executing "partials/content/card-image.html" at <$image.Resize>: nil pointer evaluating resource.Resource.Resize
Built in 281 ms
```

**Comment by h-enk** [comment, marked as answer]
Does this help: https://github.com/h-enk/doks/discussions/922#discussioncomment-3857518 ?

**Reply by akshaybajaj2280** [reply]
Thank you! Worked perfectly.
